### ISSUE TITLE
Core/Discovery and Core/resource-shape have 2 odd Core 2.0 refs

### ISSUE THREAD

**Issue by berezovskyi** [oslc-op/oslc-specs#482]
@jamsden here is the snippet from discovery.html:

```js
          OSLCSPR: {
            title: "OSLC Core 2.0 Service Provider Resources",
            href: "http://open-services.net/bin/view/Main/OslcCoreSpecification#Service_Provider_Resources",
            authors: ["S. Speicher", "D. Johnson"],
            status: "Finalized",
            publisher: "http://open-services.net",
          },
          OSLCSPV: {
            title: "OSLC Core 2.0 Selective Property Values",
            href: "http://open-services.net/bin/view/Main/OslcCoreSpecification#Selective_Property_Values",
            authors: ["S. Speicher", "D. Johnson"],
            status: "Finalized",
            publisher: "http://open-services.net",
          },
```

1. Should we have 2 references to the same spec?
2. Should we reference 2.0 specs from 3.0?

**Comment by berezovskyi** [comment]
@jamsden need your input here

**Comment by jamsden** [comment]
That's references to two anchors in the same old 2.0 spec. If these references are actually used, and they for some reason need to stay at 2.0 references (depends on the usage context), then they're ok as they are as long as they're not broken. These could be replaced with direct links to the appropriate sections in Discovery.html if they should be 3.0 references, and removed from the bibliography.

**Comment by berezovskyi** [comment]
Thank you!

https://oslc-op.github.io/oslc-specs/specs/core/discovery.html#references does not reference those two entries at all, just removed them. Same for https://oslc-op.github.io/oslc-specs/specs/core/resource-shape.html

Fixed in [`f8f5429` (#505)](https://github.com/oslc-op/oslc-specs/pull/505/commits/f8f54295c4253afb61850f5d2e5c45ddd82c5b5f)
